ci/mutmut_report.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+import json
+import os
+import csv
+
+
+def create_report():
+    mutmut_result = subprocess.check_output(["mutmut", "results"])
+    survival_rate = extract_survival_rate(mutmut_result)
+    dict_report = create_report_dict(survival_rate)
+
+    write_json_report(dict_report)
+    write_csv_report(dict_report)
+
+
+def extract_survival_rate(mutmut_result):
+    lines: list = mutmut_result.decode().split("\n")
+    before_survivors = [i for i, s in enumerate(lines) if "Survived" in s]
+    del lines[0:before_survivors[0]+1]
+
+    return sum("mutmut apply" in l for l in lines)
```

Comment:
Please consider actually making it a survival rate, which might be a better metrics, or at least changing the variable/function names so user is not confused

And let's keep an eye on this difference with last line. If you are going to extract last line anyway, might be useful extract all the data from it. And if, for example, "apply mutant" count is too different from "last line" count, report an error or so, so we would find out if we need to clear the cache or something.

Reply:
I have now done that change. I'm now basing my reporting on the 'mutmut run' command, instead of 'mutmut results'. Using the last line of the output we can calculate a percentage.